acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -119,89 +118,55 @@ def pxp_agent_config_file(host)
 
 # @param host a beaker host (to determine the correct path for the OS)
 # @param client_number which client 01-05 you want the key file for
+# @param [boolean] use_alt_path use alt file path?
```

Comment:
I think if someone new comes onto the project; this doc comment might be confusing. Perhaps

# @param [boolean] use_alt_path Use the repo's alternative set of ssl certs? Default false, use the standard set of ssl certs

(Too wordy?)